src/mito2/src/memtable/time_series.rs

```diff
@@ -199,6 +201,10 @@ impl Memtable for TimeSeriesMemtable {
         // so that we can ensure writing to memtable will succeed.
         self.update_stats(local_stats);
 
+        // update max_sequence
+        let sequence = kvs.mutation.sequence;
```

Comment:
The sequence in the `kvs` is the start sequence in the mutation. We can encapsulate a method to get the maximum sequence.
https://github.com/GreptimeTeam/greptimedb/blob/11bab0c47cce4fe09cd37763f7e1877d11178c7b/src/mito2/src/memtable/key_values.rs#L95-L104

We can also add a max sequence field to the metrics and update the sequence in `update_stats()`.